### PR TITLE
Add changelog and blog post for opam 2.1.5 release

### PR DESCRIPTION
**Pull request by tmattio**
cc @hannesm @rjbou

### CODE REVIEW

File: data/changelog/opam/2023-05-31-opam-2-1-5.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: opam 2.1.5 release
+authors: [ "Raja Boujbel" ]
+description: "Release of opam 2.1.5"
+date: "2023-05-31"
+tags: [opam, platform]
```
Comment (rjbou) [inline data/changelog/opam/2023-05-31-opam-2-1-5.md]:
```suggestion
tags: [opam, platform, release]
```

**Comment by rjbou** [comment]
Oh, I was about to open a PR, thanks! I'll open one only for opam 2.1.4 then.
We are used to update the blog posts with the discuss link, do you want to wait for its publication before merging, or edit afterwards ?

**Comment by tmattio** [comment]
> We are used to update the blog posts with the discuss link, do you want to wait for its publication before merging, or edit afterwards?

Posting on discuss first and updating the PR sounds reasonable to me 🙂

**Comment by rjbou** [comment]
I thought you wanted to wait for the discuss link and update the PR?

**Comment by tmattio** [comment]
Ugh, sorry about that, I thought you meant for upcoming releases/changelogs, not this one. I see now that the link to the post is for 2.1.3 🙈  Let's update it after you've published the Discuss post in that case

**Comment by rjbou** [comment]
No worries :)

ocaml.org update is much much faster than opam.ocaml.org one ^^

**Comment by rjbou** [comment]
In changelog entry, it is done on purpose to not keep links to issues/PRs ?

**Comment by tmattio** [comment]
>  In changelog entry, it is done on purpose to not keep links to issues/PRs ?

No actually let's remove it since you already include the changelog in the post! (I'll make a PR)

**Comment by hannesm** [comment]
Dear Madam or Sir,

I just re-read the blog post Reynir and myself wrote on ocaml.org, and to my surprise, even though there are some "authors" in the markdown, from the blog itself there's no indication who wrote it.I find this lack of attribution confusing, and would appreciate if you render the "authors" onto the website.

I personally would prefer to attribute the changelog entries to those who wrote them as well, but your mileage may vary.


See the screenshot below.

Screenshot 2023-06-27 at 22-24-22 Security in opam's cache handling (before 2 1 5)

Please also note that on the fine opam.ocaml.org/blog site there's attribution.

**Comment by rjbou** [comment]
I agree! I think too that keep visible authorship is important. Someone take time to write the post, let not remove that information. Besides, for changelog, it let the possibility for each team to choose who to set as author: the team, someone, several people, etc.

**Comment by tmattio** [comment]
Completely agree - that was not intentional: I added support for OCaml.org-hosted planet blog posts quickly to unblock us, but we should improve it and adding the authors seems important. I created an issue here to track this: https://github.com/ocaml/ocaml.org/issues/1349

**Comment by hannesm** [comment]
Dear @tmattio, thanks for your reply. I hope you find some time to work on it, it shouldn't be too hard.

**Comment by tmattio** [comment]
It shouldn't be too hard no, a PR for it would be welcome if you want to help, we always welcome contributions.

**Pull request merged**